vortex-file/src/tests.rs

```diff
@@ -552,8 +552,13 @@ async fn test_with_indices_simple() {
     let expected_numbers_split: Vec<Vec<i16>> = (0..5).map(|_| (0_i16..100).collect()).collect();
     let expected_array = StructArray::from_fields(&[(
         "numbers",
-        ChunkedArray::from_iter(expected_numbers_split.iter().cloned().map(ArrayData::from))
-            .into_array(),
+        ChunkedArray::from_iter(
+            expected_numbers_split
+                .iter()
+                .cloned()
+                .map(ArrayData::from_iter),
+        )
+        .into_array(),
```

Comment:
you want from vec for numbers

Reply:
I fixed it so it's `FromIterator<Option<$P>>` and `From<Vec<$P>>`, so most efficient for both possibilities